timescaledb/how-to-guides/backup-and-restore/pg-dump-and-restore.md

```diff
@@ -126,7 +132,25 @@ sure the new PostgreSQL instance has the same extension version as the original
 database before you perform the restore. After the data is restored, you can
 upgrade the version of TimescaleDB.
 
+### Troubleshoot errors when running pg_dump
+You might see the following errors when running `pg_dump`. You can safely ignore
+these. Your hypertable data is still copied.
+
+```bash
+pg_dump: warning: there are circular foreign-key constraints on this table:
+pg_dump:   hypertable
+pg_dump: You might not be able to restore the dump without using --disable-triggers or temporarily dropping the constraints.
+pg_dump: Consider using a full dump instead of a --data-only dump to avoid this problem.
+pg_dump: warning: there are circular foreign-key constraints on this table:
+pg_dump:   chunk
+pg_dump: You might not be able to restore the dump without using --disable-triggers or temporarily dropping the constraints.
+pg_dump: Consider using a full dump instead of a --data-only dump to avoid this problem.
+pg_dump: NOTICE:  hypertable data are in the chunks, no data will be copied
+DETAIL:  Data for hypertables are stored in the chunks of a hypertable so COPY TO of a hypertable will not copy any data.
+HINT:  Use "COPY (SELECT * FROM &lt;hypertable&gt;) TO ..." to copy all data in hypertable, or copy each chunk individually.
```

Comment:
```suggestion
pg_dump: NOTICE:  hypertable data are in the chunks, no data will be copied
DETAIL:  Data for hypertables are stored in the chunks of a hypertable so COPY TO of a hypertable will not copy any data.
HINT:  Use "COPY (SELECT * FROM &lt;hypertable&gt;) TO ..." to copy all data in hypertable, or copy each chunk individually.
```

Reply:
Just that bit of the error message should do because the rest are specific to the flags that you would specify in the pg_dump command